doc/radosgw/s3_objects_dedup.rst

```diff
@@ -85,6 +87,22 @@ If they are, we proceed with the deduplication:
 - copying the manifest from the source to the target.
 - removing all tail-objects on the target.
 
+***************
+Split Head Mode
+***************
+Dedup code can split the head object into 2 objects
+
+- one with attributes and no data and
+- a new tail-object with only data.
+
+The new-tail object will be deduped (unlike the head objects which can't be dedup)
```

Comment:
It looks like "which can't be dedup" might be more clear as in "which can't be deduped". Also, full stop at the end.  
In the previous line it's "a new tail-object" but in here "new-tail object", is the hyphenation different on purpose?